mtp_noms_ops/apps/security/forms/check.py

```diff
@@ -184,11 +189,11 @@ class AcceptOrRejectCheckForm(GARequestErrorReportingMixin, forms.Form):
     """
     CheckForm for accepting or rejecting a check.
     """
-
     fiu_action = forms.CharField(max_length=10)
+
     accept_further_details = forms.CharField(
         required=False,
-        label=CHECK_DETAIL_FORM_MAPPING['decision_reason'],
+        label=_('Give further details (optional)'),
```

Comment:
This is nice, one less level of indirection/confusion 👍

Reply:
some labels weren't used anywhere else; i.e. not looked up dynamically based on variables nor from api responses. it seemed clearer to separate out the rejection reasons (which need to be dynamically looked up based on api responses) from static form labels